src/Service/Program.cs

```diff
@@ -27,6 +27,13 @@ public class Program
 
         public static void Main(string[] args)
         {
+            // Compare current version of DAB with latest (non-rc) version in NuGet. 
```

Comment:
Would this change work for the docker scenario too?

Reply:
If it fails to reach NuGet it does not emit a warning.